{
  "connError": "Ocurri√≥ un error, porfavor intentelo nuevamente."
}